<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Screen</class>
 <widget class="QDialog" name="Screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="headLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Photos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="privewButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>570</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>View</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>570</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="pictureName">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>560</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
